jobs/models.py

```diff
@@ -101,38 +112,9 @@ class Meetup(models.Model):
         help_text="If this is a recurring meetup/event, please enter a start date.\
             Date format: DD/MM/YYYY"
     )
-    reviewer = models.ForeignKey(
-        User,
-        related_name="meetups",
-        blank=True,
-        null=True,
-        on_delete=models.SET_NULL,
-    )
-    review_status = models.BooleanField(
-        default=False,
-        help_text="Check if reviewed",
-    )
-    reviewers_comment = models.TextField(blank=True, null=True,)
-    ready_to_publish = models.BooleanField(default=False)
-    published_date = models.DateTimeField(blank=True, null=True)
-    created = models.DateTimeField(auto_now_add=True)
-    expiration_date = models.DateField(
-        blank=True,
-        null=True,
-        help_text="Automatically is set 60 days from posting. You can "
```

Comment:
Remove the word "is" from the help text (not sure why github is only showing this as deleted - it's line 34 in the new code)